test/README.md

```diff
@@ -34,4 +117,78 @@ The Test mode discovery workflow is as follows:
  - **Desired config**: the desired mode is selected at this point. The choices are Discovery, OC, BC and DualNICBC 
  - **PTP configuration**: if OC, BC or DualNICBC modes are selected, a valid default configuration is configured automatically. The output of the configuration is 1 or 2 ptpconfig objects in the openshift-ptp. If Discovery mode is selected, this configuration step is skipped.
```

Comment:
Does this work for DUAL NIC BC ?

Reply:
Dual NIC BC is not fully implemented yet, I will update the readme

Reply:
fixed